### PR TITLE
Zero Changes Implemented

### PR DESCRIPTION
**Pull request by bazexe**
Implemented the counter changes for Zero.

### CODE REVIEW

File: src/main/create-operator-json/operators/wamai.js [hunk 1]
```diff
@@ -16,5 +16,6 @@ wamai.addCounterNode(operatorId.nomad, counterType.hard,  "Wamai's MAG-NET Syste
 wamai.addCounterNode(operatorId.zofia, counterType.hard,  "Wamai's MAG-NET System pulls Zofia's Concussion and Impact Grenades.");
 wamai.addCounterNode(operatorId.gridlock, counterType.minor,  "Wamai's MAG-NET System pulls Gridlock's Trax Stinger but will not destroy it.");
 wamai.addCounterNode(operatorId.kali, counterType.hard,  "Wamai's MAG-NET System pulls Kali's Explosion Lances.");
+wamai.addCounterNode(operatorId.zero, counterType.hard, "Wamai's MAG-NET System will grab Zero's cameras out of the air and destroy them");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/wamai.js:19]: Fullstop/period needed at the end

File: src/main/create-operator-json/operators/zero.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+import counterType from '../core/counterTypeEnums';
+import operatorId from '../core/operatorIdEnum';
+import Operator from '../core/operatorFactory';
+import r6operators from "r6operators";
+
+let zero = new Operator(r6operators.zero, operatorId.zero, "Operation Shadow Legacy");
+
+zero.addCounterNode(operatorId.mira, counterType.hard, "Mira's black mirrors can be destroyed by Zero's zap.");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/zero.js:10]: Potentially change all of these to "...by Zero's Argus camera zap."?

File: src/main/scripts/imageLocation.js [hunk 3]
```diff
@@ -56,9 +56,10 @@ class ImageLocation {
             'Wamai': 'img/' + fileFormat + '/wamai.' + fileFormat,
             'Warden': 'img/' + fileFormat + '/warden.' + fileFormat,
             'Ying': 'img/' + fileFormat + '/ying.' + fileFormat,
+	    'Zero': 'img/' + fileFormat + '/zero.' + fileFormat,
```
Comment (BurkusCat) [inline src/main/scripts/imageLocation.js:59]: I think the indentation needs changed slightly for this

**Review by BurkusCat — CHANGES_REQUESTED** [review]
Looks really good @bazexe ! There are only a few small changes needed :)

**Comment by BurkusCat** [comment]
Quick question, did you test these changes out @bazexe ? I think the first step of this guide may need to be run as well: https://github.com/BurkusCat/r6-operator-counters/wiki/How-to%3A-Add-a-new-operator Otherwise, I wouldn't expect Zero's icon to show up.

**Comment by bazexe** [comment]
Yes I tested them. I checked the R6operator Icons and Zero had already been
added to the icons. I tested it on my ubuntu virtual machine and was able
to get the icon to show up along with the relationships. I will implement
the changes you said as well.

Regards,
Arbaaz Syed

**Comment by BurkusCat** [comment]
Hi @bazexe, how's it going? I just wanted to check if you were still interested in making these changes :) No pressure if you don't want to!

Hope you are keeping well!
- Burkus

**Comment by BurkusCat** [comment]
Thank you @bazexe for your contribution! @Trufflegator took over the changes you made here and made the finishing touches in this PR: #205

Hope you are well!

**Pull request closed without merging**